src/wp-includes/taxonomy.php

```diff
@@ -1426,6 +1426,22 @@ function update_termmeta_cache( $term_ids ) {
 	return update_meta_cache( 'term', $term_ids );
 }
 
+
+/**
+ * Queues term id for lazy-loading of term meta.
+ *
+ * @since 6.3.0
+ *
+ * @param array $term_ids List of term IDs.
+ */
+function wp_lazyload_term_meta( array $term_ids ) {
+	if ( ! $term_ids ) {
```

Comment:
Maybe, not a strong feeling.

```suggestion
	if ( empty( $term_ids ) ) {
```